app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts_list = @user.recent_posts
+  end
```

Comment:
The Index method is supposed to show all users, this allows current user to all other user's posts.

- [ ] Kindly update index method